rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/OneScanCovidRDTBarcodeFactoryTest.java

```diff
@@ -0,0 +1,139 @@
+package io.ona.rdt.widget;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.widget.RelativeLayout;
+
+import com.rengwuxian.materialedittext.MaterialEditText;
+import com.rey.material.widget.Button;
+import com.vijay.jsonwizard.activities.JsonFormActivity;
+import com.vijay.jsonwizard.domain.WidgetArgs;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import java.util.Date;
+
+import io.ona.rdt.R;
+import io.ona.rdt.application.RDTApplication;
+import io.ona.rdt.fragment.RDTJsonFormFragment;
+import io.ona.rdt.util.Constants;
+import io.ona.rdt.util.StepStateConfig;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.isNull;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({RDTJsonFormFragment.class, Bundle.class, RDTApplication.class, LayoutInflater.class})
+public class OneScanCovidRDTBarcodeFactoryTest {
```

Comment:
This test will undergo quite some refactor when it implements this https://github.com/onaio/rdt-standard/pull/463/commits/3be90ebdfadd9191c4b7cdcf2c1ba65db4ad7347#r485404727.

References to Powermock will need to be stripped out and leave Robolectric and Mockito. Powermock and Robolectric have been known not to play nice together.

Reply:
fixed.